src/mono/Directory.Build.props

```diff
@@ -6,17 +6,17 @@
   
   <!-- Set default Configuration and Platform -->
   <PropertyGroup>
-    <OSGroup Condition="'$(OSGroup)' == '' and '$([MSBuild]::IsOSPlatform(Windows))' == 'true'">Windows_NT</OSGroup>
-    <OSGroup Condition="'$(OSGroup)' == '' and '$([MSBuild]::IsOSPlatform(Linux))' == 'true'">Linux</OSGroup>
-    <OSGroup Condition="'$(OSGroup)' == '' and '$([MSBuild]::IsOSPlatform(OSX))' == 'true'">OSX</OSGroup>
+    <TargetOS Condition="'$(TargetOS)' == '' and '$([MSBuild]::IsOSPlatform(Windows))' == 'true'">Windows_NT</TargetOS>
+    <TargetOS Condition="'$(TargetOS)' == '' and '$([MSBuild]::IsOSPlatform(Linux))' == 'true'">Linux</TargetOS>
+    <TargetOS Condition="'$(TargetOS)' == '' and '$([MSBuild]::IsOSPlatform(OSX))' == 'true'">OSX</TargetOS>
```

Comment:
You can remove these properties and the Configuration property below in L17. They are already defined here: https://github.com/dotnet/runtime/blob/master/eng/Configurations.props